ferrocene/doc/safety-manual/src/scope.rst

```diff
@@ -31,6 +31,14 @@ This qualification is restricted to the following environment:
      - :target:`x86_64-unknown-linux-gnu`
      - ``core``, ``alloc``, ``std``, ``proc_macro``, ``test``
 
+   * - :target:`x86_64-unknown-linux-gnu`
+     - :target:`x86_64-pc-nto-qnx710`
+     - ``core``, ``alloc``
+
+   * - :target:`x86_64-unknown-linux-gnu`
+     - :target:`aarch64-unknown-nto-qnx710`
+     - ``core``, ``alloc``
```

Comment:
These targets also include `std`, `proc_macro`, and `test`.

Reply:
Oh! Yes, good catch.